src/test/java/org/broadinstitute/consent/pact/sam/SamPactTests.java

```diff
@@ -17,189 +33,272 @@
-                .headers(jsonHeaders)
-                .body(USER_STATUS.toString())
-
-                // Reject Terms of Service:
-                .given(" DELETE Sam Terms of Service")
-                .uponReceiving(" DELETE Request: " + ServicesConfiguration.REGISTER_TOS_PATH)
-                .path("/" + ServicesConfiguration.REGISTER_TOS_PATH)
-                .method("DELETE")
-                .willRespondWith()
-                .status(HttpStatusCodes.STATUS_CODE_OK)
-                .headers(jsonHeaders)
-                .body(USER_STATUS.toString())
-                .toPact();
-    }
+  private static final UserStatusDiagnostics USER_STATUS_DIAGNOSTICS =
+      new UserStatusDiagnostics()
+          .setAdminEnabled(true)
+          .setEnabled(true)
+          .setInAllUsersGroup(true)
+          .setInGoogleProxyGroup(true)
+          .setTosAccepted(true);
+
+  private static final UserStatus USER_STATUS =
+      new UserStatus()
+          .setUserInfo(new UserInfo()
+              .setUserEmail(USER_STATUS_INFO.getUserEmail())
+              .setUserSubjectId(USER_STATUS_INFO.getUserSubjectId()))
+          .setEnabled(new Enabled()
+              .setAllUsersGroup(USER_STATUS_DIAGNOSTICS.getInAllUsersGroup())
+              .setLdap(true)
+              .setGoogle(USER_STATUS_DIAGNOSTICS.getInGoogleProxyGroup()));
+
+  private final Map<String, String> JSON_HEADERS = Map.of(
+      HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON,
+      "Authentication", "Bearer: auth-token");
+
+  private final Map<String, String> TEXT_PLAIN_HEADERS = Map.of(
+      HttpHeaders.CONTENT_TYPE, MediaType.TEXT_PLAIN,
+      "Authentication", "Bearer: auth-token");
+
+  private SamDAO samDAO;
+
+  @BeforeAll
+  public static void beforeClass() {
+    System.setProperty("pact_do_not_track", "true");
+    System.setProperty("pact.writer.overwrite", "true");
+  }
 
-    @Test
-    @PactVerification(PROVIDER_NAME)
-    public void testSamContracts() throws Exception {
-        initSamDAO(mockProvider.getMockServer());
-        AuthUser authUser = new AuthUser();
-        authUser.setAuthToken("auth-token");
+  @BeforeEach
+  public void setUp(MockServer mockServer) {
+    assertThat(mockServer, is(notNullValue()));
+  }
 
-        List<ResourceType> types = samDAO.getResourceTypes(authUser);
-        assertNotNull(types);
-        assertFalse(types.isEmpty());
+  private void initSamDAO(MockServer mockServer) {
+    ServicesConfiguration config = new ServicesConfiguration();
+    config.setSamUrl(mockServer.getUrl() + "/");
+    samDAO = new SamDAO(config);
+  }
 
-        UserStatusInfo info = samDAO.getRegistrationInfo(authUser);
-        assertNotNull(info);
+  // Pacts
+  // RequestResponsePact methods define an expectation that Consent has of Sam. This expectation
+  // will be published to a Pact Broker so that if our expectation differs from what is published,
+  // we'll be notified via GitHub Action runs.
```

Comment:
```suggestion
  /* Pacts
   * RequestResponsePact methods define an expectation that Consent has of Sam. This expectation
   * will be published to a Pact Broker so that if our expectation differs from what is published,
   * we'll be notified via GitHub Action runs.
   */
```